scripts/async_profiler_build_shared.sh

```diff
@@ -5,11 +5,12 @@
 #
 set -euo pipefail
 
-VERSION=v2.9g5
-GIT_REV="60cbc128a752c657e8a97300784686e08c8dbe9d"
+VERSION=add_method_modifiers
+GIT_REV="664d04f18da2205b4f34ce422ec144dd2974906e"
 
-git clone --depth 1 -b "$VERSION" https://github.com/Granulate/async-profiler.git && cd async-profiler && git reset --hard "$GIT_REV"
-make all
+git clone --depth 1 -b "$VERSION" https://github.com/mpozniak95/async-profiler.git && cd async-profiler && git reset --hard "$GIT_REV"
+
+make all CXXFLAGS="-O3 -fno-omit-frame-pointer -fvisibility=hidden -std=c++11"
```

Comment:
I had to add `-std=c++11` because build was failing on centos glibc builder. It seems that by default centos gcc compile with c++98 standard. I am not sure if that change is ok?

Reply:
Ideally we should not make wide changes in the build such as modifying default flags. I'm not sure if it has other side effects on AP's built code. Can you do it without? i.e if the problem is `const std::vector<std::pair<int, std::string>> access_flags` then use a different construct and not a vector of pairs?

Reply:
I prefer the changes we do on upstream to remain local, i.e when the includemm option is not given, the behavior should be exact. If we change the global build flags, it's maybe not exact anymore :/